### PR TITLE
added console.log

### PR DESCRIPTION
**Pull request by benhall847**
added comments and console.log

MIGHT NOT WORK
DONT FIRE ME PLEASE BOSS

@tjcooke

### CODE REVIEW

File: MYSTUFF.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// THIS IS MY COMMENT
```
Comment (tjcooke) [inline MYSTUFF.js:1]: IS IT REALLY?

Comment (benhall847) [reply MYSTUFF.js:1]: ITS MINE STAY BACK!

Comment (tjcooke) [reply MYSTUFF.js:1]: BUT THIS IS MY REPO BRO

Comment (benhall847) [reply MYSTUFF.js:1]: YOU JUST DONT UNDERSTAND THE CODE

Comment (tjcooke) [reply MYSTUFF.js:1]: BUT IT'S A COMMENT.

**Comment by benhall847** [comment]
DONT MERGE BRO!

**Comment by benhall847** [comment]
NOOOOOO!

**Pull request merged**